lib/monitoring/logging-actor.js

```diff
@@ -0,0 +1,41 @@
+const {
+  LogEvent,
+  AbstractLoggingEngine,
+  logNothing
+} = require('./logging-engine');
+
+class MessageContextLoggingEngine extends AbstractLoggingEngine {
+  constructor (engine, actorPath) {
+    super();
+    this.engine = engine;
+    this.actorPath = actorPath;
+  }
+
+  log (logEvent) {
+    logEvent = new LogEvent(
+      logEvent.level,
+      logEvent.category,
+      this.actorPath.toString().concat(': ', logEvent.message),
+      logEvent.properties,
+      logEvent.metrics
+    );
+    this.engine.log(logEvent);
+  }
+}
+
+const prepareContextForLogging = (actor, ctx) => {
```

Comment:
I'd be happier if you moved this maybe into the base `actor.js` file with the rest of the context stuff.

Reply:
Ok, no problem.